buildkite/bazelci.py

```diff
@@ -955,7 +955,7 @@ def execute_bazel_build(bazel_binary, platform, flags, targets, bep_file, incomp
         else:
             aggregated_flags += remote_caching_flags(platform)
     aggregated_flags += flags
-    aggregated_flags += incompatible_flags
+    aggregated_flags += (incompatible_flags or [])
```

Comment:
Nit (I don't like the no-op, even if it's a one-liner):
if incompatible_flags:
  aggregated_flags += incompatible_flags

Reply:
Done~